service/src/main/java/bio/terra/cbas/dependencies/sam/SamService.java

```diff
@@ -119,15 +110,11 @@ public Result checkHealth() {
     }
   }
 
-  public boolean hasReadPermission() {
-    return hasPermission(READ_ACTION);
-  }
-
-  public boolean hasWritePermission() {
-    return hasPermission(WRITE_ACTION);
+  public boolean hasReadPermission(BearerToken userToken) {
+    return hasPermission(READ_ACTION, getSamUser(userToken), userToken);
   }
 
-  public boolean hasComputePermission() {
```

Comment:
I removed this because the only place it was used was in tests. All the APIs either currently check for read or write access only.